### PR TITLE
[Fix-6166][task-plugin]Fix Python Task Execution Error

### LINKED ISSUE #6166: [Bug] [task-plugin] Python Task execute error
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

In the newest dev version, python task is not migrated complete. When I execute python task in workflow, ds throws NullPointerException.

### What you expected to happen

Python task execute correctly.

### How to reproduce

- Add a python task into work flow.
- Type simple script, just like`print('Hello')`
- Then execute the work flow

### Anything else

_No response_

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mgsky1**
fix #6166

### CODE REVIEW

File: dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java [hunk 1]
```diff
@@ -95,8 +100,8 @@ public void setCommand(String command) {
     public void handle() throws Exception {
         try {
             //    construct process
+            this.command = buildCommand();
             TaskResponse taskResponse = pythonCommandExecutor.run(command);
```
Comment (CalvinKirs) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: Good job.
It seems we need deelete org.apache.dolphinscheduler.spi.task.AbstractTask#setCommand, it is better to use local variables.

Comment (mgsky1) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: Did you mean it is better to use local variables like `pyCommand` but NOT `this.command`?

Comment (CalvinKirs) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]:
```
String command=buildCommand(); 
```
you can refer to the ShellTask.

Comment (mgsky1) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: got it~

Comment (mgsky1) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: The code has been updated

Comment (CalvinKirs) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: also need to delete AbstractTask#setCommand

Comment (mgsky1) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java]: AbstractTask#setCommand has been deleted

**Comment by CalvinKirs** [comment]
Thank you very much for your contribution and look forward to your next contribution. If you have any questions, please discuss with me.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

**Comment by mgsky1** [comment]
> considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

Hi, I am a student from [Summer 2021](https://summer.iscas.ac.cn/#/?lang=en). I am in charge of adding blocking task. Dev branch is in 2.0 architecture now. It is  challenge for me to migrate the code from old version to newer version within limit time.😂

**Comment by CalvinKirs** [comment]
> > considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.
> 
> Hi, I am a student from [Summer 2021](https://summer.iscas.ac.cn/#/?lang=en). I am in charge of adding blocking task. Dev branch is in 2.0 architecture now. It is challenge for me to migrate the code from old version to newer version within limit time.😂

We still have 1 weeks until final evaluation. 

We can give extension to complete if mentor are happy with progress you made during coding periods.

**Pull request merged**